packages/gradient-block/src/helpers/colors.ts

```diff
@@ -0,0 +1,46 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+export const rgba2hex = (rgba: string, forceRemoveAlpha = false) => {
```

Comment:
Add tests for the whole file, and you can rename this function to « rgbaStringToHexString »
And the file can be moved to « shared »